great_expectations/checkpoint/actions.py

```diff
@@ -895,16 +897,19 @@ def _run(
         if self._using_cloud_context and expectation_suite_identifier:
             expectation_suite_ge_cloud_id = expectation_suite_identifier.id
 
-        return_val = self.target_store.set(
+        store_set_return_value = self.target_store.set(
             validation_result_suite_identifier,
             validation_result_suite,
             checkpoint_id=checkpoint_ge_cloud_id,
             expectation_suite_id=expectation_suite_ge_cloud_id,
         )
         if self._using_cloud_context:
-            return_val: GXCloudResourceRef
-            new_ge_cloud_id = return_val.id
+            store_set_return_value: GXCloudResourceRef
+            new_ge_cloud_id = store_set_return_value.id
             validation_result_suite_identifier.id = new_ge_cloud_id
+            return store_set_return_value
```

Comment:
Would this _run() method be simpler to read if cloud vs non cloud is broken out into two separate private methods? Something like (params excluded for brevity):
```
def _run(...):
    if self._using_cloud_context:
        return self._cloud_run(...)
    else:
        return self._non_cloud_run(...)
```